### PR TITLE
Optimise print format with css rather than rendering a separate templ…

### PR DESCRIPTION
**Pull request by ppettitau**
…ate.

Note - printing from html appears to be very buggy with various issues across browsers. Chrome ignores page-break-inside and will split sentences horizontally across pages. Firefox is better and appears to work as expected however will cut off the document randomly when page-break-inside is being used. 

This fix is purely an improvement on existing functionality and is definitely not perfect. We could extend on this in the future by implementing printing to pdf using another library/package e.g. pdfmake, however this would take significantly longer to implement than the current changes.

Sample print outputs:

[chrome_ang_f18.pdf](https://github.com/eresearchqut/trrf/files/7393112/chrome_ang_f18.pdf)
[firefox_ang_f18.pdf](https://github.com/eresearchqut/trrf/files/7393113/firefox_ang_f18.pdf)

### CODE REVIEW

File: rdrf/rdrf/templates/rdrf_cdes/form.html [hunk 1]
```diff
@@ -272,9 +272,9 @@
 		</a>
 
 		{% if show_print_button %}
```
Comment (ScriptSmith) [inline rdrf/rdrf/templates/rdrf_cdes/form.html:274]: This PR should remove the form print view, or there should be a follow-up issue to do so later.

**Review by ScriptSmith — CHANGES_REQUESTED** [review]
I didn't find a way to make it break consistently either :(

File: rdrf/rdrf/templates/rdrf_cdes/form.html (hunk 1 above)
Comment (ppettitau) [reply rdrf/rdrf/templates/rdrf_cdes/form.html:274]: It seems to be used by other views e.g. custom_consent_form, generic_patient, patient_edit, which is why I didn't remove it.

Comment (ScriptSmith) [reply rdrf/rdrf/templates/rdrf_cdes/form.html:274]: I don't think it's actually being used in those places

custom_consent_form:
```django
		{% if False and show_print_button %}
			<a class="btn btn-secondary" href="{% url 'rdrf:registry_form_print' registry_code form_name patient_id  %}" target="_blank">
				<span class="fa fa-print" aria-hidden="true"></span> {% trans 'Print' %}
			</a>
		{% endif %}
```

That will never render

generic_patient:
```django
		{% if show_print_button %}
			<a class="btn btn-secondary" href="{% url 'registry_form_print' registry_code form_name patient_id %}" target="_blank">
				<span class="fa fa-print" aria-hidden="true"></span> {% trans "Print" %}
			</a>
		{% endif %}
```

`show_print_button` is never set to `True` as far as I can tell, and the places `PatientFormMixin` is used don't have a `form_name` to reverse 

patient_edit:
```django
		{% if show_print_button %}
			<a class="btn btn-secondary" href="{% url 'registry_form_print' registry_code form_name patient_id %}" target="_blank">
				<span class="fa fa-print" aria-hidden="true"></span> {% trans "Print" %}
			</a>
		{% endif %}
```
Doesn't work for same reason as above

So should be removable, but please confirm

**Review by ScriptSmith — APPROVED** [review]

**Pull request merged**